modules/cnf-topology-aware-lifecycle-operator-platform-update.adoc

```diff
@@ -0,0 +1,182 @@
++
+The `PolicyGenTemplate` CR generates two policies:
+
+`du-upgrade-platform-upgrade-prep`:: This policy does the preparation work for the platform update. It creates the `ConfigMap` CR for the desired release image signature, creates the image content source of the mirrored release image repository, and updates the cluster version with the desired update channel and the update graph reachable by the spoke cluster in the disconnected environment.
+
+`du-upgrade-platform-upgrade`:: This is the policy used to perform platform upgrade.
+
+.. Add the `du-upgrade.yaml` to the `kustomization.yaml` located in the ZTP git repository for the `PolicyGenTemplate` CRs and push the changes to the git repository.
++
+ArgoCD pulls the changes from the git repository and generates the policies on the Hub cluster.
+
+.. Check the created policies by running the following command:
++
+[source,terminal]
+----
+$ oc get policies -A | grep platform-upgrade
+----
+
+. Apply the required update resources before starting the platform update with TALO.
+
+.. Save the content of the `platform-upgrade-prep` `ClusterUpgradeGroup` CR with the `du-upgrade-platform-upgrade-prep` policy and the target spoke clusters to the `cgu-platform-upgrade-prep.yml` file.
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: cgu-platform-upgrade-prep
+  namespace: default
+spec:
+  managedPolicies:
+  - du-upgrade-platform-upgrade-prep
+  clusters:
+  - spoke1
+  remediationStrategy:
+    maxConcurrency: 1
+  enable: true
+----
+
+.. Apply the policy to the Hub cluster by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f cgu-platform-upgrade-prep.yml
+----
+
+.. Monitor the update process and wait for completion. Then ensure that the policy becomes compliant by running the following command:
++
+[source,terminal]
+----
+$ oc get policies --all-namespaces
+----
+
+. Pre-cache the images for the platform update.
+.. Save the content of the platform update `ClusterGroupUpgrade` CR with the `du-upgrade-platform-upgrade` policy and the target clusters to the `cgu-platform-upgrade.yml` file. This `ClusterGroupUpgrade` CR is configured to only perform the pre-caching job without the actual cluster update.
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: cgu-platform-upgrade
+  namespace: default
+spec:
+  managedPolicies:
+  - du-upgrade-platform-upgrade
+  preCaching: true
+  clusters:
+  - spoke1
+  remediationStrategy:
+    maxConcurrency: 1
+  enable: false
+----
+
+.. Apply the CR to the Hub cluster by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f cgu-platform-upgrade-prep.yml
+----
+
+.. Monitor the date process and wait for the pre-caching to complete. Check the status of pre-caching by running the following command on the spoke cluster:
++
+[source,terminal]
+----
+$ oc get jobs,pods -n openshift-talo-pre-cache
+----
```

Comment:
```suggestion
----
$ oc get cgu cgu-platform-upgrade -o jsonpath='{.status.precaching.status}'
----
```
This will list the pre-caching status per cluster.